### PR TITLE
Iteration on the initial Course Syllabus

### PR DESCRIPTION
**Pull request by drphilmarshall**
Not yet ready for review! But here's where I'm addressing my own review comments in PR #125: 
* [x] Try splitting session 1 in two, to Basics followed by Getting Started
* [ ] Add links to missing resources (esp DIA)
* [x] Remove projects and start issues instead - and then link issues back to syllabus.

I'll add more to-do's to the list above as I go, and then request a review from Alex when I'm done.

### CODE REVIEW

**Comment by drphilmarshall** [comment]
OK @kadrlica I worked my way through sessions 1-5, adding links to a few more resources, doing some moving around of topics, and so on. I started two new project opportunity issues based on the gaps in the syllabus, #129 (Visualization: RGB cutouts) and #131 (Source detection: Explain everything). See what you think - I'm hoping this might be enough to get us started on Friday, when we could carry on and make some more issues, perhaps with the help of anyone looking for a new project.

File: Syllabus.md [hunk 1]
```diff
@@ -192,7 +205,7 @@ We will explore LSST data visualization in a bit more detail. This session will
   + [Source measurement tutorial](https://pipelines.lsst.io/getting-started/photometry.html)
   + [Kron.ipynb](https://github.com/RobertLuptonTheGood/notebooks/blob/2eeee8b9fe35077387485e488c965f1ea3d39418/Demos/Kron.ipynb)
 
-<a name="astrometry"></a>(9) Astrometric calibration
+<a name="astrometry"></a>10. Astrometric calibration
```
Comment (kadrlica) [inline Syllabus.md]: Both the Source/Object measurement and Astrometric calibration are labeled section "10".

**Comment by kadrlica** [comment]
@drphilmarshall I'm not sure how to do the joint merge of this PR with #125. What are best practices?

**Comment by drphilmarshall** [comment]
Thanks @kadrlica ! I fixed the numbering, and completed the job of removing the projects to the issue list. I'll merge this PR, since its approved and I'm the owner; then I'll approve your PR for you to tweak and merge.

**Pull request merged**